streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -483,6 +487,16 @@ public static ConfigDef configDef() {
      */
     public StreamsConfig(final Map<?, ?> props) {
         super(CONFIG, props);
+        eosEnabled = EXACTLY_ONCE.equals(getString(PROCESSING_GUARANTEE_CONFIG));
+
+        if (eosEnabled && !props.containsKey(COMMIT_INTERVAL_MS_CONFIG)) {
+            values.put(COMMIT_INTERVAL_MS_CONFIG, EOS_DEFAULT_COMMIT_INTERVAL_MS);
+        }
+
+        if (eosEnabled && getInt(NUM_STANDBY_REPLICAS_CONFIG) > 0) {
```

Comment:
I guess it was too late in the night when I added this... (the thought was that we cannot keep the replicas in sync with regard to transactional boundaries -- but of course we can... we read committed data only and reply to "end-of-log" before resuming).